src/analyses/call_graph.cpp

```diff
@@ -27,6 +27,50 @@ call_grapht::call_grapht(const goto_functionst &goto_functions)
   }
 }
 
+call_grapht::call_grapht(const symbol_tablet &symbol_table)
+{
+  for(const auto &id_and_symbol : symbol_table.symbols)
+  {
+    if(id_and_symbol.second.type.id()==ID_code &&
+       !id_and_symbol.second.is_type)
+    {
+      add(id_and_symbol.first, id_and_symbol.second.value);
+    }
+  }
+}
+
+void call_grapht::add_possible_callee(
```

Comment:
Actually I see you've added them to the header. As much as I prefer this, I think we are currently sticking with them in the .cpp file.